### PR TITLE
Addressing dependency issues

### PR DESCRIPTION
**Pull request by michelleinez**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,10 +5,10 @@
...
   "license": "Apache-2.0",
   "dependencies": {
-    "@blueprintjs/core": "^3.34.1",
-    "@material-ui/core": "^3.9.3",
-    "@material-ui/icons": "^3.0.2",
-    "@sentry/browser": "^5.26.0",
+    "@blueprintjs/core": "^3.35.0",
+    "@material-ui/core": "^4.11.0",
+    "@material-ui/icons": "^4.9.1",
+    "@sentry/browser": "^5.27.2",
```
Comment (natesulat) [inline package.json:11]: Can we remove Sentry?

**Comment by natesulat** [comment]
React linting is causing the build to fail - want to address it in this PR or save it for another one?

File: package.json (hunk 1 above)
Comment (michelleinez) [reply package.json]: Done

**Review by natesulat — APPROVED** [review]

**Review by natesulat — CHANGES_REQUESTED** [review]
Sorry, realized - need to remove the import statement for Sentry in the code

**Pull request merged**